### PR TITLE
add piphero 1.0.0

### PR DESCRIPTION
**Pull request by kiliankoe**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [ ] `brew audit --new-cask <cask>` worked successfully.

`brew audit --new-cask piphero` complains that the GitHub repository is not notable enough, but the maintainer apparently uses the GitHub repo only for publishing releases, not any source of the app. It's not linked from the app's website except via the download links, so it's probably not intended to be of note.

- [x] `brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

File: Casks/p/piphero.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+cask "piphero" do
+  version "1.0.0"
+  sha256 "3f1af01daa88ced60751f58e7c6379a45fc9807a1b0dc53c2639147ce984c6ff"
+
+  url "https://github.com/pipheroapp/downloads/releases/download/v#{version}/PiPHero-#{version}-mac-arm64.dmg",
```
Comment (SMillerDev) [inline Casks/p/piphero.rb:5]: No intel version?

**Review by SMillerDev — COMMENTED** [review]
Please add a [`zap` stanza](https://docs.brew.sh/Cask-Cookbook#stanza-zap) here. The [CreateZap](https://github.com/nrlquaker/homebrew-createzap) tool can help you with this.

**Comment by kiliankoe** [comment]
Thanks for the feedback @SMillerDev! I added the zap stanza and intel download (totally missed that)!

**Comment by muescha** [comment]
> Error: GitHub repository not notable enough (<30 forks, <30 watchers and <75 stars)

I will throw this numbers in the ring:

- 113 Upvotes at Product Hunt: https://www.producthunt.com/products/piphero
- Downloads https://github.com/pipheroapp/downloads/releases
  - PiPHero-1.0.0-mac-arm64.dmg 601 downloads
  - PiPHero-1.0.0-mac-x64.dmg 303 downloads

**Review by p-linnane — APPROVED** [review]

**Pull request merged**